.vscode/settings.json

```diff
@@ -1,3 +1,15 @@
 {
-    "behave-vsc.featuresPath": "${workspaceFolder}/trustpoint/features"
+    "behave-vsc.featuresPath": "${workspaceFolder}/trustpoint/features",
+
+    // Tell ruff-vscode to prefer your environment over its bundle
+    "ruff.importStrategy": "fromEnvironment",
+
+    "mypy-type-checker.importStrategy": "fromEnvironment",
+    
+    "ruff.interpreter": [
+        "${workspaceFolder}/.venv/bin/python"  // on Unix/macOS
```

Comment:
This obviously does not work on windows and there is no way to distinguish based on the OS vscode is started in.
I don't see this as a huge issue, but would personally include both path versions, but commented out.